llvm/lib/Target/AMDGPU/Utils/AMDGPUBaseInfo.h

```diff
@@ -1344,6 +1344,10 @@ bool isIntrinsicSourceOfDivergence(unsigned IntrID);
 /// \returns true if the intrinsic is uniform
 bool isIntrinsicAlwaysUniform(unsigned IntrID);
 
+/// Lds size is defined internally in terms of blocks
+/// Lds DW granularity is used to calculate size settings for PAL metadata 3.0+
```

Comment:
nit: rewrite this to \returns describe what the value actually means

Reply:
Done